packages/genesys-spark-components/src/components/beta/gux-calendar-beta/components/single/gux-calendar.tsx

```diff
@@ -161,6 +165,13 @@ export class GuxCalendar {
     }
   }
 
+  private isValidDate(date: Date): boolean {
+    return (
+      (this.minValue && date.getTime() <= this.minValue.getTime()) ||
+      (this.maxValue && date.getTime() > this.maxValue.getTime())
+    );
+  }
+
```

Comment:
I think this is wrong. This will return `true` for an invalid date not a valid one.

Reply:
good catch. let me rework the code.